pkg/kube/eve-bridge/eve-bridge.go

```diff
@@ -0,0 +1,501 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"net/rpc/jsonrpc"
+	"os"
+	"strings"
+
+	"gopkg.in/natefinch/lumberjack.v2"
+
+	"github.com/containernetworking/cni/pkg/invoke"
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types"
+	v1 "github.com/containernetworking/cni/pkg/types/100"
+	"github.com/containernetworking/cni/pkg/version"
+	"github.com/containernetworking/plugins/pkg/ipam"
+	bv "github.com/containernetworking/plugins/pkg/utils/buildversion"
+	"github.com/lf-edge/eve/pkg/kube/cnirpc"
+)
+
+const (
+	eveKubeNamespace       = "eve-kube-app"
+	zedrouterRPCSocketPath = "/run/zedrouter/rpc.sock"
+	primaryIfName          = "eth0"
+	clusterSvcIPRange      = "10.43.0.0/16"
+	vmiPodNamePrefix       = "virt-launcher-"
+)
+
+const (
+	logfileDir    = "/persist/newlog/kube/"
+	logfile       = logfileDir + "eve-bridge.log"
+	logMaxSize    = 100 // 100 Mbytes in size
+	logMaxBackups = 3   // old log files to retain
+	logMaxAge     = 30  // days to retain old log files
+)
+
+var logFile *lumberjack.Logger
+
+// EnvArgs encapsulates CNI_ARGS used by eve-bridge plugin.
+type EnvArgs struct {
+	types.CommonArgs
+	// Note that these variables are intentionally in the "screaming snake case".
+	// They must match the syntax of the environmental variables defined by the CNI spec.
+	// We have to ignore complains from the linter here.
+	MAC               types.UnmarshallableString
+	K8S_POD_NAME      types.UnmarshallableString
```

Comment:
Is there some comment syntax to make revive shut up for these two lines?

Reply:
Assuming yetus doesn't have some other override syntax, see [comment directives](https://github.com/mgechev/revive/blob/master/README.md#comment-directives)

I believe this would work, but I haven't tried:

```go
        //revive:disable:var-naming
	MAC               types.UnmarshallableString
	K8S_POD_NAME      types.UnmarshallableString
        //revive:enable:var-naming
```

Reply:
Yes, that works, thanks @deitch!